lib/src/functions/meta.dart

```diff
@@ -60,6 +63,22 @@ final global = UnmodifiableListView([
   })
 ]);
 
+/// The definitions of Sass introspection functions that are only available from
+/// the `sass:meta` module, not as global functions.
+final local = UnmodifiableListView([
+  _function("calc-name", r"$calc", (arguments) {
```

Comment:
should it `calc-name` or `calculation-name` as the value type has been changed to `calculation` to avoid confusion with the fact that not all calculations are for `calc()` ?

Reply:
That's something we probably should have talked through in the design process :sweat_smile:. I think it's fine though to keep this name short, since "calc" can also be read as short for "calculation".

Reply:
> That's something we probably should have talked through in the design process

I agree. But that's something I missed when re-reading the proposal after the renaming of the type from `calc` to `calculation`. It is still before to discuss that now than after landing the feature in a release.